app/controllers/concerns/session_path_concern.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+module SessionPathConcern
+  extend ActiveSupport::Concern
+
+  def transition_checker_new_session_path(**params)
+    "sign-in?#{params.compact.to_query}"
+  end
+
+  def transition_checker_new_session_callback_path(**params)
+    "sign-in/callback?#{params.compact.to_query}"
+  end
+
+  def transition_checker_end_session_path(**params)
+    "sign-out?#{params.compact.to_query}"
+  end
+
+  def transition_checker_new_session_url(**params)
+    "#{Services.frontend}/#{transition_checker_new_session_path(**params)}"
+  end
+
+  def transition_checker_new_session_callback_url(**params)
+    "#{Services.frontend}/#{transition_checker_new_session_path(**params)}"
+  end
+
+  def transition_checker_end_session_url(**params)
+    "#{Services.frontend}/#{transition_checker_new_session_path(**params)}"
+  end
```

Comment:
These `_path`s aren't used outside of this concern, so I think they should be inlined into the `_url`s.  Also you've used the `transition_checker_new_session_path` in all of the `_url`s.

Reply:
yep fair, I'll remove those, I was thinking I might need those, but guess I didn't!